core/lib/dal/src/consensus_dal.rs

```diff
@@ -1,13 +1,19 @@
+use anyhow::Context as _;
+use zksync_consensus_roles::validator;
 use zksync_consensus_storage::ReplicaState;
+use zksync_types::{Address, MiniblockNumber};
 
+pub use crate::models::storage_sync::Payload;
 use crate::StorageProcessor;
 
+/// Storage access methods for `zksync_core::consensus` module.
 #[derive(Debug)]
 pub struct ConsensusDal<'a, 'c> {
     pub storage: &'a mut StorageProcessor<'c>,
 }
 
 impl ConsensusDal<'_, '_> {
+    /// Fetch the current BFT replica state.
```

Comment:
Bikeshedding: Though there is no formal guide right now, we most often write doc-comments in the 3rd form (e.g. "Fetches" instead of "Fetch"), and I see that you also use this form below. Feel free to ignore this comment, but if you'll be touching this file otherwise, you can make it easier for me to publish all the stuff on crates.io later :)